src/pngwriter.cc

```diff
@@ -765,6 +765,14 @@ int pngwriter::read(int x, int y, int colour) const
 
 	if(bit_depth_ == 16)
 	  {
+	     /* In these cases *256 is correct, because what we actually are
```

Comment:
I know it's fantastic, isn't it? ;) I was just wondering since line 1 of this block was different then the rest